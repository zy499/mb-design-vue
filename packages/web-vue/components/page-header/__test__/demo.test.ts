import demoTest from '../../../scripts/demo-test';

demoTest('page-header');
