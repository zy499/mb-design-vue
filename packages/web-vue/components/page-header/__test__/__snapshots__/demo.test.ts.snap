// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<page-header> demo: render [basic] correctly 1`] = `
"<div class=\\"mb-page-header\\">
  <div class=\\"mb-page-header-wrapper\\">
    <!--v-if-->
    <div class=\\"mb-page-header-header\\"><span class=\\"mb-page-header-main\\"><span class=\\"mb-icon-hover mb-page-header-icon-hover mb-page-header-back-btn\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-left\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M32 8.4 16.444 23.956 32 39.513\\"></path></svg></span><span class=\\"mb-page-header-title\\">MbDesign</span><span class=\\"mb-page-header-divider\\"></span><span class=\\"mb-page-header-subtitle\\">MbDesign Vue 1.0</span></span><span class=\\"mb-page-header-extra\\"><span class=\\"mb-radio-group-button mb-radio-group-size-undefined mb-radio-group-direction-horizontal\\"><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"mini\\"><span class=\\"mb-radio-button-content\\">Mini</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"small\\"><span class=\\"mb-radio-button-content\\">Small</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"large\\"><span class=\\"mb-radio-button-content\\">Large</span></label></span></span></div>
  </div>
  <!--v-if-->
</div>"
`;

exports[`<page-header> demo: render [breadcrumb] correctly 1`] = `
"<div class=\\"mb-page-header mb-page-header-with-breadcrumb\\">
  <div class=\\"mb-page-header-wrapper\\">
    <div class=\\"mb-page-header-breadcrumb\\">
      <div role=\\"list\\" class=\\"mb-breadcrumb\\">
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">Home</div>
        <div aria-hidden=\\"true\\" class=\\"mb-breadcrumb-item-separator\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-oblique-line\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"M29.506 6.502 18.493 41.498\\"></path>
          </svg></div>
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">Channel</div>
        <div aria-hidden=\\"true\\" class=\\"mb-breadcrumb-item-separator\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-oblique-line\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"M29.506 6.502 18.493 41.498\\"></path>
          </svg></div>
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">News</div>
        <!---->
      </div>
    </div>
    <div class=\\"mb-page-header-header\\"><span class=\\"mb-page-header-main\\"><!--v-if--><span class=\\"mb-page-header-title\\">MbDesign</span><span class=\\"mb-page-header-divider\\"></span><span class=\\"mb-page-header-subtitle\\">MbDesign Vue 1.0</span></span><span class=\\"mb-page-header-extra\\"><span class=\\"mb-radio-group-button mb-radio-group-size-undefined mb-radio-group-direction-horizontal\\"><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"mini\\"><span class=\\"mb-radio-button-content\\">Mini</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"small\\"><span class=\\"mb-radio-button-content\\">Small</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"large\\"><span class=\\"mb-radio-button-content\\">Large</span></label></span></span></div>
  </div>
  <!--v-if-->
</div>"
`;

exports[`<page-header> demo: render [content] correctly 1`] = `
"<div class=\\"mb-page-header mb-page-header-with-breadcrumb mb-page-header-with-content\\">
  <div class=\\"mb-page-header-wrapper\\">
    <div class=\\"mb-page-header-breadcrumb\\">
      <div role=\\"list\\" class=\\"mb-breadcrumb\\">
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">Home</div>
        <div aria-hidden=\\"true\\" class=\\"mb-breadcrumb-item-separator\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-oblique-line\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"M29.506 6.502 18.493 41.498\\"></path>
          </svg></div>
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">Channel</div>
        <div aria-hidden=\\"true\\" class=\\"mb-breadcrumb-item-separator\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-oblique-line\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"M29.506 6.502 18.493 41.498\\"></path>
          </svg></div>
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">News</div>
        <!---->
      </div>
    </div>
    <div class=\\"mb-page-header-header\\"><span class=\\"mb-page-header-main\\"><span class=\\"mb-icon-hover mb-page-header-icon-hover mb-page-header-back-btn\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-left\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M32 8.4 16.444 23.956 32 39.513\\"></path></svg></span><span class=\\"mb-page-header-title\\">MbDesign</span><span class=\\"mb-page-header-divider\\"></span><span class=\\"mb-page-header-subtitle\\">MbDesign Vue 1.0</span></span><span class=\\"mb-page-header-extra\\"><span class=\\"mb-radio-group-button mb-radio-group-size-undefined mb-radio-group-direction-horizontal\\"><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"mini\\"><span class=\\"mb-radio-button-content\\">Mini</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"small\\"><span class=\\"mb-radio-button-content\\">Small</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"large\\"><span class=\\"mb-radio-button-content\\">Large</span></label></span></span></div>
  </div>
  <div class=\\"mb-page-header-content\\">
    <p> For other uses, see Design </p>
    <p> A design is a plan or specification for the construction of an object or system or for the implementation of an activity or process, or the result of that plan or specification in the form of a prototype, product or process. The verb to design expresses the process of developing a design. In some cases, the direct construction of an object without an explicit prior plan (such as in craftwork, some engineering, coding, and graphic design) may also be considered to be a design activity. The design usually has to satisfy certain goals and constraints, may take into account aesthetic, functional, economic, or socio-political considerations, and is expected to interact with a certain environment. Major examples of designs include architectural blueprints,engineering drawings, business processes, circuit diagrams, and sewing patterns.Major examples of designs include architectural blueprints,engineering drawings, business processes, circuit diagrams, and sewing patterns. </p>
  </div>
</div>"
`;

exports[`<page-header> demo: render [transparent] correctly 1`] = `
"<div class=\\"mb-page-header mb-page-header-with-breadcrumb\\">
  <div class=\\"mb-page-header-wrapper\\">
    <div class=\\"mb-page-header-breadcrumb\\">
      <div role=\\"list\\" class=\\"mb-breadcrumb\\">
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">Home</div>
        <div aria-hidden=\\"true\\" class=\\"mb-breadcrumb-item-separator\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-oblique-line\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"M29.506 6.502 18.493 41.498\\"></path>
          </svg></div>
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">Channel</div>
        <div aria-hidden=\\"true\\" class=\\"mb-breadcrumb-item-separator\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-oblique-line\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"M29.506 6.502 18.493 41.498\\"></path>
          </svg></div>
        <div role=\\"listitem\\" class=\\"mb-breadcrumb-item\\">News</div>
        <!---->
      </div>
    </div>
    <div class=\\"mb-page-header-header\\"><span class=\\"mb-page-header-main\\"><span class=\\"mb-icon-hover mb-page-header-icon-hover mb-page-header-back-btn\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-left\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M32 8.4 16.444 23.956 32 39.513\\"></path></svg></span><span class=\\"mb-page-header-title\\">MbDesign</span><span class=\\"mb-page-header-divider\\"></span><span class=\\"mb-page-header-subtitle\\">MbDesign Vue 1.0</span></span><span class=\\"mb-page-header-extra\\"><span class=\\"mb-radio-group-button mb-radio-group-size-undefined mb-radio-group-direction-horizontal\\"><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"mini\\"><span class=\\"mb-radio-button-content\\">Mini</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"small\\"><span class=\\"mb-radio-button-content\\">Small</span></label><label class=\\"mb-radio-button\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"large\\"><span class=\\"mb-radio-button-content\\">Large</span></label></span></span></div>
  </div>
  <!--v-if-->
</div>"
`;
